enhancements/baremetal/baremetal-compact-clusters.md

```diff
@@ -0,0 +1,139 @@
+---
+title: baremetal-compact-clusters
```

Comment:
We already have "compact bare metal clusters" in so much as it's being added to 4.5 documentation and there's an e2e-metal-compac CI job that runs UPI using only three masters. Can you change the title here and in the PR description to reflect that this really means removal of bootstrap node?

Reply:
Sure, I did try to think of an alternative title but not come up with anything yet that succinctly describes the goal, e.g there is still a bootstrap node but it's running in-place on one of the masters vs an additional install dependency.

iso-bootstrap-in-place baremetal-3-node-footprint?  Suggestions welcome, naming is hard :)

Reply:
@dhellmann suggested minimise-baremetal-footprint so I'll go with that :+1: